docker/Dockerfile

```diff
@@ -175,7 +175,16 @@ RUN --mount=type=cache,target=/root/.cache/uv \
     uv pip install --system -r requirements/build.txt \
     --extra-index-url ${PYTORCH_CUDA_INDEX_BASE_URL}/cu$(echo $CUDA_VERSION | cut -d. -f1,2 | tr -d '.')
 
-COPY . .
+COPY setup.py setup.py
+COPY pyproject.toml pyproject.toml
+COPY cmake/ cmake/
+COPY csrc/ csrc/
+COPY requirements/ requirements
+COPY vllm/ vllm/
+COPY CMakeLists.txt CMakeLists.txt
+
+COPY tools/check_repo.sh tools/check_repo.sh
```

Comment:
This helps in local builds not to invalidate cached layers. i.e. not having to rebuild vllm because you changed the script to install gdrcopy

Reply:
I also like having the whole source tree copied in too.